static/i18n/en/app.json

```diff
@@ -526,9 +526,9 @@
         "Use this tool to restore your Ledger device if it has become stuck during a firmware update.",
       "desc": "Please select the state of your Ledger device.",
       "button": "Repair",
-      "generic": "Bootloader",
-      "mcuNotGenuine": "MCU not genuine",
-      "followDeviceRepair": "Follow device repair instructions"
+      "generic": "{{repairDeviceBootloader}}",
+      "mcuNotGenuine": "{{repairDeviceMcuNotGenuine}}",
+      "followDeviceRepair": "{{repairDeviceFollow}}"
```

Comment:
these 3 wordings should be removed from translation since there is no value to add them.

Reply:
I see